dev-tools.sh

```diff
@@ -33,6 +33,6 @@
 # stable version.
 #
 # Uncomment and set the Dev-Tools's commit value and commit this change.
-# export GH_COMMIT=COMMIT_SHA
+export GH_COMMIT=7f0ad2e879dddeda3d9113c9fcfde134b2a775ac
```

Comment:
remember to remove this one before merging this PR and after the dev-tools is released with your changes

Reply:
This has been removed.